airbyte-webapp/src/components/common/ApiErrorBoundary/ApiErrorBoundary.tsx

```diff
@@ -72,6 +80,15 @@ class ApiErrorBoundaryComponent extends React.Component<
     }
   }
 
+  componentDidCatch(error: { message: string; status?: number; __type?: string }) {
+    if (isFormBuildError(error)) {
+      this.props.trackError(error, {
+        id: "formBuildError",
+        connector: error.connectorId,
```

Comment:
```suggestion
        connectorDefinitionId: error.connectorId,
```
nit suggestion to make it clear that this is a connector definition ID (since connector ID usually refers to the ID of an instantiated source or destination record). 

Could you also update the field on the FormBuildError class to be `connectorDefinitionId` instead of `connectorId`?